recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.4.0rc1" %}
```

Comment:
Also, I should note, that there weren't any releases partly because Cyclus didn't build anywhere and I kvetched about it. So there *is* a bit of a boot strapping issue.